examples/virtualservice_test.yml

```diff
@@ -12,6 +12,10 @@ testCases:
         host: users.users.svc.cluster.local
         port:
           number: 80
+    headers:
```

Comment:
What does this check mean exactly? that the header needs to be set by istio for each request?

Reply:
For the given `request` defined in the test it's expected the header to be set.